### ISSUE TITLE
File explorer opening when clicking on HeroCard button

### ISSUE THREAD

**Issue by yochail** [microsoft/BotFramework-Emulator#1397]
## Version
4.3.3
Win 10 Home  10.0.17763

## Describe the bug
When clicking on a hero card with "imBack", a file explorer is opening ("This PC"), in addition to sending the postback

## To Reproduce
this message:
```json
{
  "attachmentLayout": "carousel",
  "attachments": [
    {
      "content": {
        "buttons": [],
        "images": [
          {}
        ],
        "subtitle": "Where do you want to order from?",
        "tap": {
          "type": "showImage"
        },
        "title": "Welcome, User!"
      },
      "contentType": "application/vnd.microsoft.card.hero"
    },
    {
      "content": {
        "buttons": [
          {
            "text": "Select this store",
            "title": "Select this store",
            "type": "imBack",
            "value": "Select store sajj san jose"
          }
        ],
        "images": [
          {
            "url": ""
          }
        ],
        "subtitle": "4140 North First St Ste 10 San Jose CA 95134",
        "tap": {
          "type": "showImage",
          "value": ""
        },
        "title": "sajj san jose"
      },
      "contentType": "application/vnd.microsoft.card.hero"
    },
    {
      "content": {
        "buttons": [
          {
            "text": "Select this store",
            "title": "Select this store",
            "type": "imBack",
            "value": "Select store Sajj Mountain View"
          }
        ],
        "images": [
          {
            "url": ""
          }
        ],
        "subtitle": "2580 West El Camino Real #4 Mountain View, CA 94040",
        "tap": {
          "type": "showImage",
          "value": ""
        },
        "title": "Sajj Mountain View"
      },
      "contentType": "application/vnd.microsoft.card.hero"
    },
    {
      "content": {
        "buttons": [
          {
            "text": "Select this store",
            "title": "Select this store",
            "type": "imBack",
            "value": "Select store Sajj Sunnyvale"
          }
        ],
        "images": [
          {
            "url": ""
          }
        ],
        "subtitle": "145 S Frances ST Sunnyvale, CA 94086",
        "tap": {
          "type": "showImage",
          "value": ""
        },
        "title": "Sajj Sunnyvale"
      },
      "contentType": "application/vnd.microsoft.card.hero"
    },
    {
      "content": {
        "buttons": [
          {
            "text": "Select this store",
            "title": "Select this store",
            "type": "imBack",
            "value": "Select store sajj menlo park"
          }
        ],
        "images": [
          {
            "url": ""
          }
        ],
        "subtitle": "883 Hamilton Ave, Menlo Park, CA 94025",
        "tap": {
          "type": "showImage",
          "value": ""
        },
        "title": "sajj menlo park"
      },
      "contentType": "application/vnd.microsoft.card.hero"
    },
    {
      "content": {
        "buttons": [
          {
            "text": "Select this store",
            "title": "Select this store",
            "type": "imBack",
            "value": "Select store sajj san francisco"
          }
        ],
        "images": [
          {
            "url": ""
          }
        ],
        "subtitle": "636 2nd Street, San Francisco, CA 94107",
        "tap": {
          "type": "showImage",
          "value": ""
        },
        "title": "sajj san francisco"
      },
      "contentType": "application/vnd.microsoft.card.hero"
    },
    {
      "content": {
        "buttons": [
          {
            "text": "Select this store",
            "title": "Select this store",
            "type": "imBack",
            "value": "Select store sajj rancho santa margarita"
          }
        ],
        "images": [
          {
            "url": ""
          }
        ],
        "subtitle": "22245 El Paseo, Suite A, Rancho Santa Margarita, CA 92688",
        "tap": {
          "type": "showImage",
          "value": ""
        },
        "title": "sajj rancho santa margarita"
      },
      "contentType": "application/vnd.microsoft.card.hero"
    },
    {
      "content": {
        "buttons": [
          {
            "text": "Select this store",
            "title": "Select this store",
            "type": "imBack",
            "value": "Select store sajj san jose"
          }
        ],
        "images": [
          {
            "url": ""
          }
        ],
        "subtitle": "4140 North First St Ste 10 San Jose CA 95134",
        "tap": {
          "type": "showImage",
          "value": ""
        },
        "title": "sajj san jose"
      },
      "contentType": "application/vnd.microsoft.card.hero"
    }
  ],
  "channelId": "emulator",
  "conversation": {
    "id": "c5f4a830-5072-11e9-8d51-6ba1c40e95ee|livechat"
  },
  "entities": [],
  "from": {
    "id": "cf446530-43d2-11e9-b7f8-a571966de5c1",
    "name": "Bot",
    "role": "bot"
  },
  "id": "5cad8e90-5073-11e9-8dd7-d732260bde0b",
  "localTimestamp": "2019-03-27T11:33:21+02:00",
  "locale": "",
  "recipient": {
    "id": "61309d57-169b-4858-ab31-a740e0c92b6c",
    "role": "user"
  },
  "replyToId": "1b684470-5073-11e9-8dd7-d732260bde0b",
  "serviceUrl": "http://localhost:2175",
  "showInInspector": true,
  "suggestedActions": {
    "actions": [
      {
        "title": "Send Location",
        "type": "imBack",
        "value": "Send Location"
      }
    ]
  },
  "text": "Choose from the carousel, Enter a Zip Code, city, or send your location.",
  "timestamp": "2019-03-27T09:33:21.785Z",
  "type": "message"
}
```

click on "select this store" will open file explorer (in addition for sending the postback to the bot)

## Expected behavior
only send postback to the bot

[bug]

**Comment by tonyanziano** [comment]
I was able to repro this. This is pretty strange behavior.

**Comment by yochail** [comment]
BTW I'm pretty sure the reason for that behavior is the "Tap" action firing when you click to the button (and not only when you click the card)

**Comment by tonyanziano** [comment]
@yochail this should be fixed in our 4.5 release slated for later this month. It should also be in tonight's [nightly build](https://github.com/Microsoft/botframework-emulator-nightlies/releases) released around 10:30 PM PST.
